snapshotter/utils/rpc.py

```diff
@@ -365,43 +365,42 @@ async def _async_web3_call(self, contract_function, redis_conn, from_address=Non
         )
         async def f(node_idx):
             try:
-
                 node = self._nodes[node_idx]
-                rpc_url = node.get('rpc_url')
+                rpc_url = node.get("rpc_url")
```

Comment:
Let's stick to the convention we're using in our codebase, this should be in single quotes not double. Same goes for everything else below. Also, you should follow contributor guidelines https://github.com/PowerLoom/pooler#for-contributors and always run `pre-commit` hooks.